catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchFilterVisitor.java

```diff
@@ -77,12 +78,12 @@ public OpenSearchFilterVisitor() {
     @Override
     public Object visit(Not filter, Object data) {
         Object newData;
-        NestedTypes parentNest = curNest;
+        NestedTypes parentNest = currentNest;
         logger.debug("ENTERING: NOT filter");
```

Comment:
No need to... in truth we should check with the original author(s)... there (good) might be reasons why they made them debugs... and who knows maybe they're not even needed any more... regardless it's out of scope for your ticket.